rise/static/rise.yaml

```diff
@@ -0,0 +1,216 @@
+    <strong>(effective only if <code>overlay</code> is not defined)</strong>
+  input_type: text
+  default:
+#
+- name: rise.footer
+  description: >
+    <code>footer</code>: the html text for a footer that will show up on all slides.
+    <strong>(effective only if <code>overlay</code> is not defined)</strong>
+  input_type: text
+  default:
+#
+- name: rise.backimage
+  description: >
+    <code>backimage</code>: a (relative) filename for a background image that will show up on all slides.
+    <strong>(effective only if <code>overlay</code> is not defined)</strong>
+  input_type: text
+  default:
+#
+- name: rise.overlay
+  description: >
+    <code>overlay</code>: some html text that will be inserted as-is on all pages. If this is provided,
+    the 3 previous settings (header, footer and backimage) are ignored altogether.
+  input_type: text
+  default:
+#
+# the UI elements for RISE itself
+#
+- name: rise.toolbar_icon
+  description: >
+    <code>toolbar_icon</code>: font-awesome class for the RISE button in toolbar
+  default: fa-bar-chart
+#
+- name: rise.shortcuts.slideshow
+  description: <code>rise.shortcuts.slideshow</code> shortcut to enter and exit slideshow 
+  input_type: hotkey
+  default: alt-r
+- name: rise.shortcuts.toggle-slide
+  description: <code>rise.shortcuts.toggle-slide</code> shortcut to toggle the 'slide' tag on current cell
+  input_type: hotkey
+  default: shift-i
+- name: rise.shortcuts.toggle-subslide
+  description: <code>rise.shortcuts.toggle-subslide</code> shortcut to toggle the 'subslide' tag on current cell
+  input_type: hotkey
+  default: shift-u
+- name: rise.shortcuts.toggle-fragment
+  description: <code>rise.shortcuts.toggle-fragment</code> shortcut to toggle the 'fragment' tag on current cell
+  input_type: hotkey
+  default: shift-f
+- name: rise.shortcuts.toggle-notes
+  description: <code>rise.shortcuts.toggle-notes</code> shortcut to toggle the 'notes' tag on current cell
+  input_type: hotkey
+  default: none
+- name: rise.shortcuts.toggle-skip
+  description: <code>rise.shortcuts.toggle-skip</code> shortcut to toggle the 'skip' tag on current cell
+  input_type: hotkey
+  default: none
+- name: rise.shortcuts.render-all-cells
+  description: <code>rise.shortcuts.render-all-cells</code> shortcut to put all cells in render mode
+  input_type: hotkey
+  default: none
+- name: rise.shortcuts.toggle-skip
+  description: <code>rise.shortcuts.edit-all-cells</code> shortcut to put all cells in edit mode
+  input_type: hotkey
+  default: none
+
+#
+# passed to reveal as-is
+#
+- name: rise.theme
+  description: >
+    <code>theme</code>: <strong>This setting, like all the ones below,
+    is passed directly to <a href="https://github.com/hakimel/reveal.js">
+    <code>reveal.js</code> see this doc</a></strong>.
+    <br/> theme: <a href="https://github.com/hakimel/reveal.js#theming">see a list of reveal's available native themes</a>
```

Comment:
Can you add a note here about dark themes not supported? In fact, you can use a dark theme, but you need to modify some things with a custom CSS to make it work.

Reply:
see next commit